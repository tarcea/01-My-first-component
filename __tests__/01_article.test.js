import React from 'react';
import renderer from 'react-test-renderer';
import Article from '../lib/01_article.jsx';

test('Article should produce the right HTML', () => {
  const article = renderer.create(
    <Article title="My fancy new product" body="Test it now!" />
  );

  let tree = article.toJSON();
  expect(tree).toMatchSnapshot();
});
