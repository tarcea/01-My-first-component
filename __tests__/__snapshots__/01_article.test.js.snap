// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article should produce the right HTML 1`] = `
<div
  className="article"
>
  <h2>
    My fancy new product
  </h2>
  <p>
    Test it now!
  </p>
</div>
`;
